### ISSUE TITLE
NullPointerException in PickupDragController#saveSelectedWidgetsLocationAndStyle()

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fredsa/gwt-dnd#156]
```
Does the issue occur in "quirks mode", "standards mode" or both? If you
don't know, does your HTML page contains a DOCTYPE declaration?
Happens in "standards mode" ("<!DOCTYPE html>")

What version of GWT are you using? 1.4.60? 2.0.4? Other?
2.4.0

What version of the gwt-dnd jar file or library file are you using?
3.1.0

What operating system(s) are you using? Windows? Linux? Mac?
The exception happened on "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.7; 
rv:8.0.1) Gecko/20100101 Firefox/8.0.1"

Does the issue occur in web mode, development mode (formerly "hosted
mode"), both or don't know?
Appeared in web-mode.

What browser(s) do you use? Chrome, Firefox, IE, Safari, other?
Firefox

What is the browser version (if you know) from Help->About?
8.0.1

What steps will reproduce the problem? Please attach sample code if you
can.
Don't know. The exception was reported to me through gwt-log.

What is the expected output? What do you see instead?
The following exception was reported:

Caused by: com.google.gwt.core.client.JavaScriptException: (TypeError): (LFg[e] 
= ZWh + kJg, b).c is null 
        at Unknown.Ms(StackTraceCreator.java:168) 
        at Unknown.zh(StackTraceCreator.java:421) 
        at Unknown.Sq(Throwable.java:46) 
        at Unknown.zIb(Exceptions.java:29) 
        at Unknown.fh(PickupDragController.java:435) 
        at Unknown.jh(PickupDragController.java:193) 
        at Unknown.Jg(MouseDragHandler.java:513) 
        at Unknown.Ng(MouseDragHandler.java:218) 
        at Unknown.oA(MouseMoveEvent.java:53) 
        at Unknown.NC(SimpleEventBus.java:194) 
        at Unknown.CC(HandlerManager.java:127) 
        ... 9 more 

Do you have a workaround?
Nope.

Please provide any additional information below.
This has low priority as I saw the exception only once so far. I think this is 
caused by info.initialDraggableParent beeing null. If you need any further 
details don't hesitate to contact me.
```

Original issue reported on code.google.com by `wuerting...@gmail.com` on 4 Feb 2012 at 5:00
